test/integration/generate-renderer.spec.js

```diff
@@ -53,9 +53,9 @@ describe('when a class breaks classification passed in', () => {
     })
   })
   describe('with no input data', () => {
-    it('should throw and error', () => {
+    it('should return empty object', () => {
       data = {}
-      generateRenderer.bind(this, data, options).should.throw()
+      generateRenderer(data, options).should.deepEqual({})
```

Comment:
This expectation changed after observing response of ArcGIS server under the same circumstances.